# deprecated message type, used only in deprecated messages
std_msgs/Header header
float64 latitude
float64 longitude
float64 altitude
float64[9] position_covariance
std_msgs/String sol_status
std_msgs/String pos_type
float64 undulation
int8 nb_sat_tracked
int8 nb_sat_solution